tests/common.js

```diff
@@ -49,6 +51,15 @@ const getMockDeviceEvent = app => async (...events) => {
   }, events);
 };
 
+const getWCClientMock = app => async (method, args) => {
+  return await app.client.execute(
```

Comment:
don't need to await when returning promise

Reply:
will change